raiden/tests/utils/transport.py

```diff
@@ -284,6 +286,7 @@ def make_requests_insecure():
     # Disable verification in requests by replacing the 'verify'
     # attribute with non-writable property that always returns `False`
     requests.Session.verify = property(lambda self: False, lambda self, val: None)  # type: ignore
+    urllib3.disable_warnings(InsecureRequestWarning)
```

Comment:
From my point of view, calling `make_requests_insecure()` is bold enough, so we don't need to disable the warning explicitely.